app/helpers/portal_helper.rb

```diff
@@ -61,4 +61,8 @@ def generate_article_link(portal_slug, article_slug, theme, is_plain_layout_enab
   def render_category_content(content)
     ChatwootMarkdownRenderer.new(content).render_markdown_to_plain_text
   end
+
+  def random_thumbnail_bg_color
+    ['#6D95BA', '#A4C3C3', '#E19191'].sample
```

Comment:
We can change the implementation slightly; This code will ensure that the same username always gets the same color, but different usernames are likely to get different colors.

```rb
def thumbnail_bg_color(username)
    colors = ['#6D95BA', '#A4C3C3', '#E19191']
    colors[username.length % colors.size]
end
```

Reply:
Updated with this change https://github.com/chatwoot/chatwoot/pull/8414/commits/d5510be166a4f1490b708ae84015245381ac1425